backend/s3/s3.go

```diff
@@ -2628,6 +2628,7 @@ type Options struct {
 	Region                string               `config:"region"`
 	Endpoint              string               `config:"endpoint"`
 	STSEndpoint           string               `config:"sts_endpoint"`
+	DualstackEndpoint     bool                 `config:"dualstack_endpoint"`
```

Comment:
Can we call this `use_dual_stack` since it is a bool? That would fit in better with the other options.

Reply:
> Can we call this `use_dual_stack` since it is a bool? That would fit in better with the other options.

Thanks I’ll update with the feedback and thanks for catching the deprecation.  There’s a v1 & v2 AWS api for GO and a couple other deprecations so this is one i missed.  I’ll have the updates shortly.

Reply:
Migrate to the new AWS SDK is probably something we should bite the bullet and do at some point, however that is out of scope for this PR :-)